xyz-hub-service/src/main/java/com/here/xyz/hub/rest/ApiParam.java

```diff
@@ -158,6 +157,9 @@ private static String getConvertedKey(String rawKey) {
       put("=cs=", QueryOperation.CONTAINS);
     }};
 
+    private static List<String> shortOperators = new ArrayList<>(operators.keySet());
```

Comment:
The short operators are tested in a for loop and the order of the list has a significance. E.g. testing `>` before `>=` would mean `>=` is never matched, testing `=` before `=gt=` would mean that `=gt=` is never matched.

Reply:
...this has been reflected by adding comparison on L286
if( .... || ( currentPositionOfOp == position && op.length() < shortOperator.length() )